brickschema/validate.py

```diff
@@ -0,0 +1,341 @@
+"""
+The `validate` module implements a wrapper of `pySHACL`_ to
+validate an ontology graph against the default Brick Schema constraints (called *shapes*) and user-defined shapes.
+
+.. _`pySHACL`: https://github.com/RDFLib/pySHACL
+"""
+import os
+import logging
+from rdflib import Graph, Namespace, URIRef, BNode, Literal
+from rdflib.plugins.sparql import prepareQuery
+from .namespaces import BRICK, A, RDF, RDFS, BRICK, BSH, SH, SKOS
+import pyshacl
+import io
+import pkgutil
+
+logging.basicConfig(
+    format="%(asctime)s,%(msecs)03d %(levelname)-7s [%(filename)s:%(lineno)d] %(message)s",
+    datefmt="%Y-%m-%d:%H:%M:%S",
+    level=logging.WARNING)
```

Comment:
To check: we're doing this here so that we can move the logging configuration to `brickschema/__init__.py` later?

Reply:
This is actually deadcode because we know now that the first logging.basicConfig (in inference.py) takes preference.  It will be taken away in another PR.  But if you look at the __init__ function for Validator(),  "getLogger('validate')" will stay.